### DISCUSSION TITLE
Resize a bunch of images in a folder.

### DISCUSSION THREAD

**Discussion by razvanab** [chaiNNer-org/chaiNNer#2103] [Ideas]
Resize of rescale
If there is a way to do this already, I apologize, but if not, I would like to have this option.
Thank you.

**Comment by joeyballentine** [comment]
You can use the Image File Iterator to run nodes on images in batch.

**Reply by razvanab** [reply]
Ok, I see now.
Will that respect the folder's heritage too?

**Reply by joeyballentine** [reply]
It will if you connect the "relative path" connection between the load and save nodes

**Reply by razvanab** [reply]
You mean subdirectory path ?
![image](https://github.com/chaiNNer-org/chaiNNer/assets/2854730/487af5e9-9a8a-4bc6-ade6-2c1857c82fad)

**Reply by RunDevelopment** [reply]
Yes, exactly.

**Reply by razvanab** [reply]
Ok, thanks
